docs/diagrams/wallet-lifecycle/wallet-lifecycle-1-handling-frauds.tikz

```diff
@@ -78,13 +78,13 @@
 % DEFEAT FRAUD CHALLENGE
 % Nodes
 \node[start, right=of fc_start, xshift=120mm]                    (df_start)   {\textbf{Defeat fraud challenge}};
-\node[check, below=of df_start]                                  (df_check1)  {Does the provided\\ preimage match an\\ existing, unresolved\\ challenge??};
+\node[check, below=of df_start]                                  (df_check1)  {Does the\\ provided heartbeat\\ message match an\\ existing, unresolved\\ challenge?};
```

Comment:
This is wrong - I asked to remove the double `??` but `preimage` was correct.

We use the heartbeast message only for `Defeat fraud challenge with heartbeat`. Here, we need to use preimage.